### DISCUSSION TITLE
How to set reviewers for an issue in github projects

### DISCUSSION THREAD

**Discussion by jhu7235** [community/community#40320] [New to GitHub]
I have a project in https://github.com/orgs/orgX/projects/1 and I see a field for "Reviewers". It's a reserved field, so I can't manually set it. I am wondering how that value is set. Currently, it doesn't look like it is setting.

**Comment by AlefMemTav** [comment]
See it: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review

**Comment by jfullstackdev** [comment]
from the link provided above, let me highlight this 

![image](https://user-images.githubusercontent.com/47092464/204709228-29ddd6eb-c0cc-4699-8558-a4ca95851997.png)
